src/markdown.cpp

```diff
@@ -2070,11 +2070,15 @@ void Markdown::writeOneLineHeaderOrRuler(const char *data,int size)
   }
   else // nothing interesting -> just output the line
   {
-    m_out.addStr(data,size);
+    int tmpSize = size;
+    if (data[size-1] == '\n') tmpSize--;
```

Comment:
is there a risk that size==0 here? if so we are accessing data[-1]

Reply:
I don't think this risk exists, but to be on the save side we could test at the start of the routine whether or not `size` is 0, in that case we could return immediately.

Feel free to change.

Reply:
Updated pull request loos OK to me

Reply:
made the update from within the browser 😉

Reply:
Nice that it is possible and for easy changes it is not that bad, only disadvantage is that it starts a new CI build (especially appveyor) and afterwards one again with the merge, but this is not really a doxygen problem. With a local update only the CI build is executed on the merge (rest is done locally).